biz.aQute.resolve/testdata/obr-exporter/target/.gitignore

```diff
@@ -0,0 +1 @@
+*.xml*
```

Comment:
Another symptom of incorrectly using the real Bnd workspace as the test workspace.

Reply:
DONE ✔️